HTML,CSS,JAVAScript/exercise-05/script.js

```diff
@@ -0,0 +1,244 @@
+       
+    }
+   
+    </script>
+    </html>`;
+}
+
+let tempArr = [];
+function renderPokemon(pokeData) {
+    let allPokemonContainer = document.getElementById('container2')
+
+    // let allPokemonContainer = document.getElementById('poke-container');
+    let pokeContainer = document.createElement("div") //div will be used to hold the data/details for indiviual pokemon.{}
+    pokeContainer.style.height = 370
+    pokeContainer.style.width = 50
+    pokeContainer.className = "card"
+
+    let pokeName = document.createElement('h3')
+    pokeName.innerText = "Name :" + pokeData.name
+    pokeName.className = "name"
+
+    let poke_details = document.createElement('button')
+    // poke_details.target="Details"
+    // poke_details.href="file://home/jestonfernandes/Downloads/um-exercises-main/exercise-02/index.html"
+    // poke_details.text="Load more"
+    poke_details.name = "Load More"
+    poke_details.addEventListener("click", onLoadMore)
+    poke_details.id = `#${pokeData.id}`
+    poke_details.style.height = "20px"
+    poke_details.style.width = "150px"
+    poke_details.textContent = "Click for more details"
+
+    let pokeweight = document.createElement('h4')
+    pokeweight.innerText = "\tWeight :" + pokeData.weight
+    pokeweight.className = "weight"
+    let pokeexperience = document.createElement('h4')
+    pokeexperience.name = "pke"
+    pokeexperience.class = "pkc"
+    pokeexperience.innerText = "\tBase_experience :" + pokeData.base_experience
+    let poke_hp = document.createElement('h4')
+    poke_hp.innerText = "\tHp : " + pokeData.stats[0].base_stat
+    let pokeNumber = document.createElement('p')
+    pokeNumber.innerText = `#${pokeData.id}`
+    let pokeImage = document.createElement('img')
+    pokeImage.src = pokeData.sprites.front_default
+
+
+
+    let frontImage = pokeData.sprites.front_default
+    let backImage = pokeData.sprites.back_default
+
+
+    pokeImage.addEventListener("mouseover", onHover);
+    pokeImage.addEventListener("mouseout", offHover);
+
+
+    pokeImage.height = 120
+    pokeImage.width = 130
+    pokeImage.className = "myimage"
+    pokeImage.id = `#${pokeData.id}`
+    pokeContainer.append(pokeName, pokeImage, pokeNumber, pokeweight, pokeexperience, poke_hp, poke_details);
+    //appending all details to the pokeContainer div
+    allPokemonContainer.appendChild(pokeContainer);
+
+
+    function onHover(e) {
+        console.log("onHover")
+        document.getElementById(e.target.id).src = backImage;
+        //this.src=backImage;
+    }
+
+    function onLoadMore(e) {
+        console.log("onclick")
+        console.log((e.target.id).slice(1))
+
+
+        fetchdetails((e.target.id).slice(1));
+
+
+        //this.src=backImage;
+    }
+
+    
+    function offHover(e) {
+        document.getElementById(e.target.id).src = frontImage;
+    }
+
+    let value=document.getElementById('sort')
+console.log(value.value)
+switch(value.value) {
+    case 0:
```

Comment:
it would be better to use case values such as weight, base_experience and so on, it's much more readable.